src/DetailsView/components/requirement-view.tsx

```diff
@@ -77,6 +81,52 @@ export class RequirementView extends React.Component<RequirementViewProps> {
         };
     }
 
+    private renderNextRequirementButton(
```

Comment:
This feels like it should be a separate new component. I'm also okay with this being in the next requirement button.